apps/ledger-live-mobile/src/newArch/features/WalletSync/screens/Activation/ActivationDrawer.tsx

```diff
@@ -41,11 +40,12 @@ function View({
         hasBackButton={canGoBack}
         onBack={goBackToPreviousStep}
       >
-        <Flex maxHeight={maxDrawerHeight}>
+        <Flex maxHeight={"90%"}>
```

Comment:
Why not 100%, even with 100% you still have a gap between top of screen if I'm correct

Reply:
Not on some devices (with last iphones and the top bar it doesn't work)
![image](https://github.com/user-attachments/assets/17c26fc8-9d36-49af-9fe6-519d7606ec8f)